README.md

```diff
@@ -33,11 +33,11 @@ Also see info about related [Microsoft .NET Core and ASP.NET Core Bug Bounty Pro
 
 ## Filing issues
 
-This repo should contain issues that are tied to the runtime, the class libraries and frameworks, the installation of the `dotnet` binary (sometimes known as the `muxer`) and installation of the .NET Core runtime and libraries.
+This repo should contain issues that are tied to the runtime, the class libraries and frameworks, the installation of the `dotnet` binary (sometimes known as the `muxer`) and installation of the .NET runtime and libraries.
```

Comment:
What are the rules on why "runtime" in ".NET runtime" is sometimes uppercase and sometimes lowercase?

Reply:
I do not think there is any rule. It depends on context in number of cases, e.g. ".NET Runtime and libraries" would look a bit odd in this specific case.